browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -2137,6 +2137,9 @@ export class CommentSection extends app.definitions.canvasSectionObject {
 			this.update();
 		}
 
+		var showResolved = this.map.stateChangeHandler.getItemValue('ShowResolvedAnnotations');
```

Comment:
don't have to be with `.uno`? like  in: https://github.com/CollaboraOnline/online/blob/f55c1a1403fcd350e81fdb98bd6c3a6b05cfda2b/browser/src/docdispatcher.ts#L538

Reply:
nop, `getItemValue` adds it if the command doesn't contain `.uno:` prefix.